jujubundlelib/changeset.py

```diff
@@ -218,7 +219,9 @@ def _handle_unit_placement(
             parent_record_id = 'addMachines-{}'.format(changeset.next_action())
             options = {}
             if placement.container_type:
-                options = {'containerType': placement.container_type}
+                options = {
+                    'containerType': _lxd_to_lxc(placement.container_type)
```

Comment:
Why not just:
```python
# Define Linux container types.
LXC = 'lxc'
LXD = 'lxd'
```
and then, here 
```python
if placement.container_type:
    ctype = placement.container_type
    if ctype == LXD:
        ctype = LXC
    options = {'containerType': ctype}
```
I am not sure it's worth using regexp for such a simple substitution (probably `'lxd'.replace('lxd', 'lxc')` would be faster than regexp in this case, fwiw).